README.md

```diff
@@ -7,7 +7,35 @@ Export iOS and tvOS IPA from an existing Xcode archive
+- Make sure certificates are uploaded to Bitrise's **Code Signing** tab. The right provisioning profiles are automatically downloaded from Apple as part of the automatic code signing process.
+
+To configure the Step:
+1. **Archive Path**: Specifies the archive that should be exported. The input value sets xcodebuild's `-archivePath` option.
+2. **Select a product to distribute**: Decide if an App or an App Clip IPA should be exported.
+3. **Distribution method**: Describes how Xcode should export the archive: development, app-store, ad-hoc, or enterprise.
+
+Under **Automatic code signing**:
+1. **Automatic code signing method**: Select the Apple service connection you want to use for code signing. Available options: `off` if you don't do automatic code signing, `api-key` [if you use API key authorization](https://devcenter.bitrise.io/en/accounts/connecting-to-services/connecting-to-an-apple-service-with-api-key.html), and `apple-id` [if you use Apple ID authorization](https://devcenter.bitrise.io/en/accounts/connecting-to-services/connecting-to-an-apple-service-with-apple-id.html).
+2. **Register test devices on the Apple Developer Portal**: If this input is set, the Step will register the known test devices on Bitrise from team members with the Apple Developer Portal. Note that setting this to `yes` may cause devices to be registered against your limited quantity of test devices in the Apple Developer Portal, which can only be removed once annually during your renewal window.
+3. **The minimum days the Provisioning Profile should be valid**: If this input is set to >0, the managed Provisioning Profile will be renewed if it expires within the configured number of days. Otherwise the Step renews the managed Provisioning Profile if it is expired.
+4. The **Code signing certificate URL**, the **Code signing certificate passphrase**, the **Keychain path**, and the **Keychain password** inputs are automatically populated if certificates are uploaded to Bitrise's **Code Signing** tab. If you store your files in a private repo, you can manually edit these fields.
+
+Under **IPA export configuration**:
+1. **Developer Portal team**: Add the Developer Portal team's name to use for this export. This input defaults to the team used to build the archive.
+2. **Rebuild from bitcode**: For non-App Store exports, should Xcode re-compile the app from bitcode?
+3. **Include bitcode**: For App Store exports, should the package include bitcode?
+4. **iCloud container environment**: If the app is using CloudKit, this input configures the `com.apple.developer.icloud-container-environment` entitlement. Available options vary depending on the type of provisioning profile used, but may include: `Development` and `Production`.
+5. **Export options plist content**: Specifies a `plist` file content that configures archive exporting. If not specified, the Step will auto-generate it.
```

Comment:
I have a separate PR that adds a new input called `Manage version and build number`. 

This will change the version and build number of all content in your app. If your app has an invalid number (like one that was used previously, or precedes your current build number), the step provides an option to automatically increment it to a valid number. The step does not work if export options plist content has been set.

Reply:
@shams-ahmed can you please update the step.yml with this info?